### ISSUE TITLE
Detect circumstances when the API returns an empty JSON doc

### ISSUE THREAD

**Issue by gridhead** [t0xic0der/prudence#7]
And address the same with the help of required procedures

**Comment by pedroandrade** [comment]
@t0xic0der I've got situations that JSON was empty, maybe, an error code from server? like 5XX or 4XX, we could improve checking for status code, what do you think?

**Comment by gridhead** [comment]
Yes, the error codes can act as an indicator but the situation that we are trying to deal here is kinda different. Consider a scenario when an endpoint is working just fine, the internet connection is all good and every other situation is positive - but the endpoint does not have anything to say.

For instance, events - an endpoint will return an empty JSON document when there are no ongoing events. That certainly does not mean that the endpoint is messed up or something has gone wrong. :sweat_smile:

**Comment by pedroandrade** [comment]
Yeah, you're right I just thought about errors scenarios, so, there are lot of scenarios that JSON response could be empty, so in this case, we can show a message like "Nothing to Say!", instead of our base template, what do you think?

**Comment by gridhead** [comment]
Sounds good to me.
